README.md

```diff
@@ -72,6 +72,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Keraf](#keraf) - **streaming:** JavaScript, C#, Node.js, .NET Core, React, Web Development, Browser Extension Dev
 - [Kevin Griffin](#kevin-griffin) - **streaming:** C#, .NET, ASP.NET, 3D Printing, Conference Planning, and more!
 - [KeyFramers](#KeyFramers) - **streaming:** CSS, Javascript, UI Animation, Web Development
+- [Krissy Heishman](#ninjabunny9000) - **streaming:** Python, C#, Ninja, Programming in general
```

Comment:
The content in the parentheses are IDs for anchor tags within this page. In order to link to Krissy's section properly, this needs to be `#krissy-heishman` (because it's based off the name of the header lower in the page).